lib/agent0/agent0/base/agents/eth_wallet.py

```diff
@@ -4,14 +4,17 @@
 import copy
 import logging
 from dataclasses import dataclass, field
-from typing import Any
+from typing import TYPE_CHECKING, Any
 
+from agent0.base import Quantity, TokenType
 from elfpy import check_non_zero
-from elfpy.types import Quantity, TokenType
-from elfpy.wallet.wallet_deltas import WalletDeltas
 from fixedpointmath import FixedPoint
 from hexbytes import HexBytes
 
+if TYPE_CHECKING:
+    # TODO need a base wallet delta
```

Comment:
I don't think we should leave this TODO -- this means there is a backwards dependency, where base is depending on hyperdrive. The base version is easy, it only has `balance`.